src/main/java/gov/hhs/cms/bluebutton/fhirstress/utils/BenefitIdMgr.java

```diff
@@ -0,0 +1,88 @@
+package gov.hhs.cms.bluebutton.fhirstress.utils;
+
+/**
+ * A utility method that helps manage an index into a RIF file containting
+ *
+ */
+public class BenefitIdMgr {
```

Comment:
So if each test thread uses a separate instance of this class, does that mean that all of the threads are hitting roughly the same benes in the same order? If so, that provides the system being sampled with an unrealistic caching boost/bonus. You probably want to randomize the order that benes are selected in such that it's different for each thread.

(Probably doesn't matter for the synthetic data, since it's small enough that the DB server can keep most of it hot in the cache anyways. But probably does matter in prod.)

Reply:
I think this is going to be changed anyways when/if we move to an S3 configuration.  But duly noted.